HealthcareSystem/PatientService/Service/PatientService.cs

```diff
@@ -70,5 +70,19 @@ public void UpdateMedicalInfo(string jmbg, MedicalInfoUpdateDTO medicalInfoUpdat
             patient.UpdateMedicalInfo(medicalInfoUpdate);
             _repository.Update(patient);
         }
+
+        public IEnumerable<Therapy> GetTherapiesForExamination(string jmbg, int id)
+        {
+            Patient patient = _repository.Get(jmbg);
+            IEnumerable<Therapy> therapies = patient.Therapies;
+
+            return therapies.Where(t => t.ExaminationId == id);
+        }
+        public Examination GetExamination(string jmbg, int id)
+        {
+            Patient patient = _repository.Get(jmbg);
+            IEnumerable<Examination> examinations = patient.Examinations;
+            return examinations.Single(e => e.Id == id);
+        }
```

Comment:
I feel like the logic for both of these should maybe be pushed into Patient instead?

Reply:
Applied in 18cc2d8254942669b62c63a476da24757bd1b566.